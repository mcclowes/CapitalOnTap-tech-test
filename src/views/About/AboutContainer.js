import React, { useState } from "react";
import About from "./About";
import { Loader } from "../../components/UIKit";

const AboutContainer = props => {
  // an exammple of a useful hook and loading state
  const [loading, setloading] = useState(false);

  return loading ? <Loader /> : <About {...props} />;
};

export default AboutContainer;
