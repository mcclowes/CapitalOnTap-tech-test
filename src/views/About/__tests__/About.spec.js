import About from "../About";
import React from "react";
import { shallow } from "enzyme";

const defaultProps = {};

const renderComponent = newProps => {
  const props = {
    ...defaultProps,
    ...newProps
  };

  return shallow(<About {...props} />);
};

let wrapper;

it("renders component", () => {
  wrapper = renderComponent();

  expect(wrapper).toMatchSnapshot();
});
