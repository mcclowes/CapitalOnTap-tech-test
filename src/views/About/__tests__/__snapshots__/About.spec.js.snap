// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `
<Fragment>
  <Nav />
  <csx__PageWrapper>
    <csx__Page>
      <Text>
        Made by 
        <a
          href="https://github.com/mcclowes"
        >
          @mcclowes
        </a>
      </Text>
    </csx__Page>
  </csx__PageWrapper>
</Fragment>
`;
